### DISCUSSION TITLE
从4月23日开始内存占用很高，查看进程1panel占用1g+

### DISCUSSION THREAD

**Discussion by gtjyj** [1Panel-dev/1Panel#4719] [Q&A]
<img width="723" alt="图片" src="https://github.com/1Panel-dev/1Panel/assets/25652352/a35abc26-0004-4102-b4c0-0d2b88b3524a">
1panel占用内存1g+, 重启1panel后内存占用变为100多MB，请问是否和前几天的版本升级有关系

**Comment by wanghe-fit2cloud** [comment]
> <img alt="图片" width="723" src="https://private-user-images.githubusercontent.com/25652352/325834844-a35abc26-0004-4102-b4c0-0d2b88b3524a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxMDA3NTIsIm5iZiI6MTcxNDEwMDQ1MiwicGF0aCI6Ii8yNTY1MjM1Mi8zMjU4MzQ4NDQtYTM1YWJjMjYtMDAwNC00MTAyLWI0YzAtMGQyYjg4YjM1MjRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI2VDAzMDA1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlYTU1YzBjMDExZjVkZTMwMjZlNmQ1MWY5MWZiZmQ5NTFiNzY1NGY3ZTY0NGE5MGQwZjY2MGVlMDcwZWU3YjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.voZDa9is9-EPBVhn0hkYlz_JewvybrKI0QU0uya1CTU"> 1panel占用内存1g+, 重启1panel后内存占用变为100多MB，请问是否和前几天的版本升级有关系

可能是和部分 docker client 没关闭有关系，我们本地已经复现该问题，会在下个版本尝试修复。

**Reply by gtjyj** [reply]
3q
